de.fu_berlin.inf.dpp.ui.frontend/html/src/components/BasicWidgetTestView/index.jsx

```diff
@@ -57,8 +57,8 @@ export default class BasicWidgetTestView extends React.Component {
     } else {
       value = e.target.id
     }
-    $('#button-display-text').html(value)
-    console.log(value)
+    document.querySelector('#button-display-text').innerHtml = value
```

Comment:
did the linter say that you have to change from JQuery to Vanilla JavaScript? that is weird

Reply:
okay, but just for future changes bear in mind that the STF testing requires JQuery.

Reply:
The tests are green. Otherwise, Travis would have complained already.